ang/civicase/ActivityFeed.js

```diff
@@ -432,25 +425,29 @@
        */
       function affixActivityDetailsPanel () {
         $timeout(function () {
-          $activityDetailsPanel = $element.find('.panel');
+          $activityDetailsPanel = $element.find('.civicase__activity-panel');
           $filter = $('.civicase__activity-filter');
           $feedListContainer = $('.civicase__activity-feed__list-container');
           $tabs = $('.civicase__dashboard').length > 0 ? $('.civicase__dashboard__tab-container ul.nav') : $('.civicase__case-body_tab');
           $toolbarDrawer = $('#toolbar');
 
           $activityDetailsPanel.affix({
             offset: {
-              top: $element.find('.panel').offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
+              top: $activityDetailsPanel.offset().top - ($toolbarDrawer.height() + $tabs.height() + $filter.height()),
               bottom: $($document).height() - ($feedListContainer.offset().top + $feedListContainer.height())
```

Comment:
please encapsulate this calculation
```js
$($document).height() - ($feedListContainer.offset().top + $feedListContainer.height())
```
as it's been repeated in two different places